tests/units/classes/test.php

```diff
@@ -57,6 +57,16 @@ public function __construct()
 
 			parent::__construct();
 		}
+
+		static public function staticMethod($return)
+		{
+			return $return;
+		}
+
+		static public function someOtherStaticMethod($return1, $return2, $return3)
+		{
+			return [$return1, $return2, $return3];
```

Comment:
be careful with short-array syntax, we are old-school guys and still support php 5.3 ;)